Dockerfile.build

```diff
@@ -0,0 +1,16 @@
+ARG JAVA_VERSION=11
```

Comment:
Is the name `Dockerfile.build` the accepted standard?

Reply:
No.  This file is intentionally named differently as it's used to create the build environment.  The Science Portal requires Gradle, JDK 11, and NPM, and if you like keeping your desktop clean, then this is used to build an image that can be used to build the Science Portal.

```sh
docker run --rm -ti -v $(pwd):/app -w /app image-from-dockerfile-build gradle -i clean build test
```